Source/RapyutaSimulationPlugins/Public/Core/RRCrowdROSController.h

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file RRCrowdROSController.h
+ * @brief Base Crowd ROS Controller for ARRBaseRobot descedent classes, utilizing UCrowdFollowingComponent
+ * @copyright Copyright 2020-2022 Rapyuta Robotics Co., Ltd.
+ */
+
+#pragma once
+
+#include "Core/RRCrowdAIController.h"
+
+#include "RRCrowdROSController.generated.h"
+
+class URRRobotROS2Interface;
+
+/**
+ * @brief Base Crowd ROS Controller for ARRBaseRobot descedent classes, utilizing UCrowdFollowingComponent
+ */
+UCLASS()
```

Comment:
Do we need RRCrowdAIController which has only different constructor from AIController?
Can't we merge RRCrowdAIController and RRCrowdROSController

Reply:
Yeah I thought about it also, but as suggested by the name, CrowAIController should be for common pawns or robots not using ROS only.

Reply:
Yeah I thought about it also, but as suggested by the name, CrowAIController should be only for common pawns or robots not using ROS.